cf_remote/ssh.py

```diff
@@ -174,7 +174,7 @@ def scp(file, remote, connection=None, rename=None, hide=False):
             if file == rename:
                 return 0
             print_function("Renaming '%s' -> '%s' on '%s'" % (file, rename, remote))
-            ssh_cmd(connection, "mv %s %s" % (file, rename), hide=hide)
+            ssh_cmd(connection, "mv %s %s" % (file, rename))
```

Comment:
I got an error here about unexpected hide parameter but now am not so sure... checking.